### PR TITLE
Rename config item for skipper health check

### PR DESCRIPTION
**Pull request by RomanZavodskikh**
Signed-off-by: Roman Zavodskikh <roman.zavodskikh@zalando.de>

### CODE REVIEW

**Comment by RomanZavodskikh** [comment]
:+1:

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -76,11 +76,11 @@ skipper_ingress_hpa_scale_up_max_perc: "100"
 {{if eq .Cluster.Environment "production"}}
 skipper_ingress_min_replicas: "3"
 skipper_ingress_max_replicas: "300"
-skipper_ingress_passive_health_check_options: ""
+skipper_ingress_health_check_options: ""
```
Comment (szuecs) [inline cluster/config-defaults.yaml]: we can even drop this now as we rolled this out to all clusters!

File: cluster/config-defaults.yaml [hunk 2]
```diff
@@ -76,11 +76,11 @@ skipper_ingress_hpa_scale_up_max_perc: "100"
...
 {{else}}
 skipper_ingress_min_replicas: "2"
 skipper_ingress_max_replicas: "50"
-skipper_ingress_passive_health_check_options: "period=10s,min-requests=10,max-drop-probability=0.9"
+skipper_ingress_health_check_options: "period=10s,min-requests=10,max-drop-probability=0.9"
```
Comment (szuecs) [inline cluster/config-defaults.yaml]: let's move it out of `if..else`

**Comment by RomanZavodskikh** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**